normandy/classifier/api.py

```diff
@@ -6,7 +6,8 @@
 
 
 class FetchBundle(views.APIView):
-
+    authentication_classes = []
+    permission_classes = []
```

Comment:
Does it make sense to change the defaults? I don't think our API will ever want CSRF.